bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/utils/SvnUtil.java

```diff
@@ -221,90 +222,30 @@ private static boolean setupWorkingCopy(SVNMergeUnit mergeUnit, ISvnClient clien
 			workingCopy.mkdirs();
 		}
 
-		List<Path> paths = new ArrayList<>(targetFile.size() + 1);
```

Comment:
Nice refactoring. Took me a while for review. 👍

Reply:
Hopefully, thats a good thing. :)